backend/solr/doc.py

```diff
@@ -217,3 +217,14 @@ def from_result(res: dict) -> str:
         else:
             log.debug("CreationDate is unknown.")
             return str(datetime.now())
+
+__all__ = [
+    'SolrDoc',
```

Comment:
`SolrDoc` is the only one needed from here the others are used by SolrDoc itself to parse.

Reply:
Or do you need thos all for the documentation stuff?